pkgs/top-level/python-packages.nix

```diff
@@ -1491,6 +1491,8 @@ in {
 
   boltztrap2 = callPackage ../development/python-modules/boltztrap2 { };
 
+  boolean-py = callPackage ../development/python-modules/boolean-py { };
```

Comment:
Can you please split the new packages into separate commits?

Reply:
Um, well, yes, I certainly can, even though I’m not quite sure what would be the point, given that those new packages are not particularly useful on their own 🤷‍♂.

Reply:
Thank you.